app/src/main-process/menu/build-default-menu.ts

```diff
@@ -211,7 +211,7 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
         click: emit('open-in-shell'),
       },
       {
-        label: __DARWIN__ ? 'Open in Finder' : '&Open in Explorer',
+        label: __DARWIN__ ? 'Show in Finder' : '&Show in Explorer',
```

Comment:
This conflicts with line 221
''' label: __DARWIN__ ? 'Repository Settings…' : 'Repository &settings…', '''
Can't have both map to 'S' in windows.  Maybe  Open in &Explorer